website/src/categories/components/message-dialog/message-dialog.md

```diff
@@ -10,6 +10,6 @@ tags: component
 eleventyNavigation:
   parent: Components
   key: Message dialog
-  order: 13
+  order: 16
```

Comment:
Can you please fix the order to have all components alphabetically? :)

<img width="292" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/848dd9ee-570d-489f-b990-61a660be54d0">